FogCreek/Glitch-Community
# Profuse tea: Fix project items with links in their description

## Links
https://profuse-tea.glitch.me/@producthunt
https://app.clubhouse.io/glitch/story/2828/project-items-with-links-in-their-description-do-not-ssr-properly

## GIF/Screenshots:
![image](https://user-images.githubusercontent.com/8932174/67894787-a7dfaf80-fb2f-11e9-84e3-23336383ecff.png)

## Changes:
- Project items strip links out of their description until after the first render
- Light refactoring on `AvatarBase` to remove the `type` prop
- The Glitch Team avatar (not `@Glitch`, the one that hides project members) is square like a team

## How To Test:
Try loading up the product hunt team and notice how the links in ~google-home and ~alexa-skill pop in a moment late. Try loading it up on glitch.com and notice how those two project items are stretched out for a moment.

## Feedback I'm looking for:
The issue is that nested `<a>` tags are not allowed and this pr doesn't really fix that. It just stalls rendering them until browsers handle it cleanly.
